gatsby-theme-landing-page/src/components/heading.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import * as styles from "./heading.module.css";
+
+export default function Heading({
+  children,
+  as,
+  secondary,
+  className,
+  ...rest
+}) {
+  const baseStyle = !secondary ? styles.heading : styles.secondaryHeading;
+
+  /*
+   Uses `as` prop for heading element. If not provided, defaults to 
+   h2 for standard and h3 for secondary
+  */
+  const Component = as || secondary ? "h3" : "h2";
+  return children ? (
+    <Component className={`${baseStyle} ${className}`} {...rest}>
+      {children}
+    </Component>
+  ) : null;
+}
```

Comment:
Not blocking – an alternative component API for this could have one base Heading component and a second Subhead or SecondaryHeading component that sets a default `as` and includes styles; would me less logic in the components and the ability the shadow the two components somewhat independently

Reply:
Yeah, agreed. Let's implement that